.pre-commit-config.yaml

```diff
@@ -29,7 +29,7 @@ repos:
           - flake8-unused-arguments
 
   - repo: https://github.com/codespell-project/codespell
-    rev: v2.2.5
+    rev: v2.2.4
```

Comment:
sorry about that! I indeed used development env and didn't notice such "divergence". Filed https://github.com/codespell-project/codespell/issues/2843 and meanwhile let's have this fixed. Thank you!